_posts/2023-04-05-#27255.md

```diff
@@ -10,14 +10,44 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Background and prior work
+
+_Miniscript_ is a language for writing (a subset of) Bitcoin Scripts in a structured way, enabling analysis, composition, generic signing and more. _Miniscript_ support in Bitcoin Core has been an ongoing effort, with watch-only support for miniscript descriptors added in `v24.0.1`. For background:
+
+* [https://bitcoin.sipa.be/miniscript](https://bitcoin.sipa.be/miniscript) gives an overview of the miniscript langauge
+* [#24147](https://github.com/bitcoin/bitcoin/pull/24147) provides the rationale and initiale backbone needed for miniscript support in Bitcoin Core
+* [#24148](https://github.com/bitcoin/bitcoin/pull/24148) adds watch-only support for miniscript descriptors
+* [#24149](https://github.com/bitcoin/bitcoin/pull/24149) adds signing support for miniscript descriptors
+
+### TapMiniscript
+
+While [_Tapscript_](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki) shares most operations with legacy Bitcoin Script, there are a few notable differences. As such, _TapMiniscript_ has been a project to support tapscript in miniscript. For background:
+
+* [This gist](https://gist.github.com/sipa/06c5c844df155d4e5044c2c8cac9c05e) contains the initial design and discussion on supporting miniscript in tapscript
+* [#134](https://github.com/sipa/miniscript/pull/134/) adds tapscript support in miniscript
+
+### Putting it all together
+
+Putting it all together, this PR adds support for Tapscript in Miniscript in Bitcoin Core and makes Miniscript available inside `tr()` descriptors, both with watching and signing support.
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
+1. In your own words, what are some of the differences between Segwit v0 Script (Bitcoin Script with the Segwit Soft Fork rules) and Segwit v1 script (Tapscript)?
```

Comment:
Maybe let's ask the question in the context of Miniscript?

```suggestion
1. In your own words, what are some of the differences between Segwit v0 Script (Bitcoin Script with the Segwit Soft Fork rules) and Tapscript rules that may be relevant to Miniscript? Why?
```

Also nit i removed Segwit v1 because Tapscript is only defined for a specific leaf, in the future another Segwit v1 output with another leaf version used may have different Script semantics.